pkgs/development/python-modules/scipy/default.nix

```diff
@@ -1,31 +1,48 @@
-{lib, fetchPypi, python, buildPythonPackage, gfortran, nose, pytest, numpy}:
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, gfortran
+, nose
+, pytest
+, numpy
+, cython
+, pybind11
+}:
 
 buildPythonPackage rec {
   pname = "scipy";
-  version = "1.3.3";
+  version = "1.4.1";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "64bf4e8ae0db2d42b58477817f648d81e77f0b381d0ea4427385bba3f959380a";
+  src = fetchFromGitHub {
```

Comment:
This breaks eval for `python2Packages.scipy`:
https://github.com/NixOS/nixpkgs/blob/286450b33528d951959a7cc5d085c266e3a5b043/pkgs/top-level/python-packages.nix#L5161-L5164